code/modules/pda/app.dm

```diff
@@ -67,7 +66,9 @@
 	if(pda.current_app)
 		pda.current_app.stop()
 	pda.current_app = src
-	return 1
+	if(!pda.silent)
```

Comment:
the reason this has to be boilerplated so much is because some apps create sounds, and we don't want sounds to overlap between the beeping and say, the honk sound the clown plays---there's also some instances where we do/do not want the sound to play within the app, again, to avoid overlap (sending messages, for instance).

the app code is not very atomic or OOP---and I'm not about to refeactor the entirety of it just to make playing sounds a bit cleaner.